### ISSUE TITLE
Settings for Training

### ISSUE THREAD

**Issue by jeighfrost** [TAMU-VITA/EnlightenGAN#53]
Hi! Thank you for sharing the EnlightenGAN. I just want to ask some questions cause I'm kinda new to this. How am I able to find the best setting for training? I used the same training dataset that you provided and used the same settings for training (100 epoch with learning rate of 0.0001 followed by another 100 epochs with learning rate linearly decayed to zero) but the trained model seemed to differ a lot from the pretrained model. What factors should I consider about this? Which of which parameters should I will change to get a close or a better result with the pretrained model?

**Comment by yueruchen** [comment]
Have you used exactly same setting as I provided in the script?

**Comment by jeighfrost** [comment]
Yes, i didn't change anything on the script and I used the same set of training images in the link that was provided

**Comment by yueruchen** [comment]
Did you use pytorch 0.3.1 and batchsize 32? Could you provide some results?

**Comment by yueruchen** [comment]
If no reply I will close this issue, feel free to contact me if you meet new problems.
